### PR TITLE
Update configs

### PR DESCRIPTION
**Pull request by alex-taxiera**
- updated travis config
- updated eslintrc
  - must use curly brackets
  - must space array literals
  - max line length 80 enforced

- files updated to reflect new style

### CODE REVIEW

File: src/QueryBuilder.js [hunk 1]
```diff
@@ -35,7 +35,11 @@ class QueryBuilder {
   }
   _clean (data) {
     const x = { ...data }
-    for (const y in x) if (x[y] instanceof Object) x[y] = JSON.stringify(x[y])
+    for (const y in x) {
+      if (x[y] instanceof Object) {
```
Comment (mets11rap) [inline src/QueryBuilder.js]: Is it possible to remove the curly braces here?

File: src/QueryBuilder.js [hunk 2]
```diff
@@ -61,12 +65,19 @@ class QueryBuilder {
...
+      }
       return this._knex.schema.createTable(name, (table) => {
-        if (this._knex.client.config.client === 'mysql') table.charset('utf8')
+        if (this._knex.client.config.client === 'mysql') {
+          table.charset('utf8')
+        }
         for (const column of columns) {
           if (!table[column.type]) {
-            throw Error(`'${column.name}' uses '${column.type}' which is not an existing type.`)
+            throw Error(
```
Comment (mets11rap) [inline src/QueryBuilder.js]: Strings spread across multiple lines are a bit weird on the eyes

File: src/QueryBuilder.test.js [hunk 3]
```diff
@@ -60,68 +71,100 @@ test('count: table with one entry', async (t) => {
 
 test('createTable: primary', (t) => {
   const tableName = 'newprimary'
-  return t.context.QueryBuilder.createTable({ name: tableName, columns: [{ name: 'column1', type: 'string', primary: true }] })
-    .then(async (rows) => t.is(rows.name, tableName))
+  return t.context.QueryBuilder.createTable({
+    name: tableName,
+    columns: [ { name: 'column1', type: 'string', primary: true } ]
+  }).then(async (rows) => t.is(rows.name, tableName))
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: When methods are chained, we put them all on the next lines. Put this `then` on the next line and indent.

File: src/QueryBuilder.test.js [hunk 4]
```diff
@@ -60,68 +71,100 @@ test('count: table with one entry', async (t) => {
...
 test('createTable: primary default', (t) => {
   const tableName = 'newprimarydefault'
-  return t.context.QueryBuilder.createTable({ name: tableName, columns: [{ name: 'column1', type: 'string', primary: true, default: 'yolo' }] })
-    .then(async (rows) => t.is(rows.name, tableName))
+  return t.context.QueryBuilder.createTable({
+    name: tableName,
+    columns: [
+      { name: 'column1', type: 'string', primary: true, default: 'yolo' }
+    ]
+  }).then(async (rows) => t.is(rows.name, tableName))
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: Put on next line

File: src/QueryBuilder.test.js [hunk 5]
```diff
@@ -60,68 +71,100 @@ test('count: table with one entry', async (t) => {
...
 })
 
 test('createTable: normal default', (t) => {
   const tableName = 'newdefault'
-  return t.context.QueryBuilder.createTable({ name: tableName, columns: [{ name: 'column1', type: 'string', default: 'yolo' }] })
-    .then(async (rows) => t.is(rows.name, tableName))
+  return t.context.QueryBuilder.createTable({
+    name: tableName,
+    columns: [ { name: 'column1', type: 'string', default: 'yolo' } ]
+  }).then(async (rows) => t.is(rows.name, tableName))
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: Put on next line

File: src/QueryBuilder.test.js [hunk 6]
```diff
@@ -60,68 +71,100 @@ test('count: table with one entry', async (t) => {
...
 })
 
 test('createTable: normal', (t) => {
   const tableName = 'newnormal'
-  return t.context.QueryBuilder.createTable({ name: tableName, columns: [{ name: 'column1', type: 'string' }] })
-    .then(async (rows) => t.is(rows.name, tableName))
+  return t.context.QueryBuilder.createTable({
+    name: tableName,
+    columns: [ { name: 'column1', type: 'string' } ]
+  }).then(async (rows) => t.is(rows.name, tableName))
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: Put on next line

File: src/QueryBuilder.test.js [hunk 7]
```diff
@@ -135,60 +178,67 @@ test('dropTable', (t) => {
...
-        column: 'integer',
-        amount: 2,
-        where: {
-          key: 'increment this'
-        }
-      }).then((rows) => t.is(rows.find((row) => row.key === 'increment this').integer, 5))
-    })
+  return t.context.QueryBuilder.insert({
+    table, data: { key: 'increment this', integer: 3 }
+  }).then(() => t.context.QueryBuilder.increment({
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: Put these `then`s on new lines indented.

File: src/QueryBuilder.test.js [hunk 8]
```diff
@@ -135,60 +178,67 @@ test('dropTable', (t) => {
...
-          }).then((rows) => {
-            t.is(rows.find((row) => row.key === 'increment this').integer, 5)
-            t.is(rows.find((row) => row.key === 'dont increment').integer, 3)
-          })
-        })
-    })
+  return t.context.QueryBuilder.insert({
+    table,
+    data: { key: 'increment this', integer: 3 }
+  }).then(() => t.context.QueryBuilder.insert({
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: Put these `then`s on newlines indented.

File: src/QueryBuilder.test.js [hunk 9]
```diff
@@ -199,29 +249,41 @@ test('insert', (t) => {
...
-    .then(() => {
-      return t.context.QueryBuilder.raw('SELECT * FROM ' + table)
-        .then((res) => {
-          if (Array.isArray(res[0])) res = res[0]
-          t.truthy(res.find((row) => row.key === 'findthisinraw'))
-        })
+  return t.context.QueryBuilder.insert({
+    table,
+    data: { key: 'findthisinraw' }
+  }).then(() => t.context.QueryBuilder.raw('SELECT * FROM ' + table))
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: Put this `then` on the next line.

File: src/QueryBuilder.test.js [hunk 10]
```diff
@@ -232,16 +294,21 @@ test('renameTable', (t) => {
...
 
 test('select/insert: an array as a string', (t) => {
   const table = t.context.tables.general
-  return t.context.QueryBuilder.insert({ table, data: { key: 'arr', string: ['123', '456'] } })
-    .then(() => t.context.QueryBuilder.select({ table, where: { key: 'arr' } }))
-    .then((rows) => t.deepEqual(rows[0].string, ['123', '456']))
+  return t.context.QueryBuilder.insert({
+    table,
+    data: { key: 'arr', string: [ '123', '456' ] }
+  }).then(() => t.context.QueryBuilder.select({ table, where: { key: 'arr' } }))
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: Put this `then` on the next line

File: src/QueryBuilder.js (hunk 1 above)
Comment (alex-taxiera) [reply src/QueryBuilder.js]: not with the current design, maybe it can be revised

File: src/QueryBuilder.js (hunk 2 above)
Comment (alex-taxiera) [reply src/QueryBuilder.js]: I think it looks good when they're wrapped in parens like this.

File: src/QueryBuilder.test.js (hunk 3 above)
Comment (alex-taxiera) [reply src/QueryBuilder.test.js]: Okay, i'll enable that rule

**Pull request merged**